migrations/challenges/sync-all-challenges.js

```diff
@@ -0,0 +1,40 @@
+import Bluebird from 'Bluebird';
+
+import { model as Challenges } from '../../website/server/models/challenge';
+import { model as User } from '../../website/server/models/user';
+
+async function syncChallengeToMembers (challenges) {
+  challenges.forEach(async function (challenge) {

```

Comment:
this will still do all the challenges in parallel and it'll return at line 19 immediately because forEach doesn't work with await, it'll kickstart the async function and then return